templates/etc/netplan/config.yaml.j2

```diff
@@ -0,0 +1,27 @@
+network:
+  ethernets:
+{% for interface in vm_unique_net_interfaces %}
+    {{ interface.name }}:
+      addresses:
+{% if (interface.ip4 is defined) %}
+        - {{ interface.ip4 }}/{{ interface.netmask4 }}
+{% endif %}
+{% if (interface.ip6 is defined) %}
+        - {{ interface.ip6 }}/{{ interface.netmask6 }}
+{% endif %}
+{% if (interface.ip4 is defined) %}
```

Comment:
here should it be `{% if (interface.gw4 is defined) %}` ?

Reply:
It was to see if you were paying attention.